parlai/tasks/woz/build.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Download and build the data if it does not exist.
```

Comment:
This comment seems a bit outdated or misplaced.

Reply:
Do you mean "Download and build the data if it does not exist." ? That's at the top of all of the build.py files I think

Reply:
okay nvm i guess